README.md

```diff
@@ -1,9 +1,21 @@
-# Filecoin (cpp-filecoin)  
+# Filecoin (cpp-filecoin)
 
 > C++17 implementation of blockchain based digital storage
 
 Filecoin is a decentralized protocol described in [spec](https://filecoin-project.github.io/specs/)
 
+## Minimal hardware requirements
+### Node minimal parameters:
+**Hard disk space**: at least 200 GB  
+**RAM**: 8 GB  
+**OS**: Linux(Ubuntu), MacOS, other operating systems builds are not tested yet, so running on them may be unstable
```

Comment:
Other OS are not supported, some functions are not implemented for them.

Reply:
```suggestion
**OS**: Linux(Ubuntu), MacOS, other operating systems builds are not tested yet, so running on them may be unstable.
```

Reply:
fixed